### PR TITLE
【inplace api】batch add inplace api paddle.log_, paddle.i0_, paddle.nn.functional.leaky_relu_...

### PR DESCRIPTION
**Pull request by GGBond8488**
### PR types
Others

### PR changes
APIs

### Description
Pcard-72375
batch add inplace api as follow

> paddle.nn.functional.hardtanh_
paddle.nn.functional.leaky_relu_
paddle.nn.functional. thresholded_relu_
paddle.digamma_
paddle.trunc_
paddle.frac_
paddle.i0_
paddle.lgamma_
paddle.log_
paddle.log10_
paddle.log1p_
paddle.log2_
paddle.logit_
paddle.neg_
paddle.polygamma_
paddle.tril_
paddle.triu_
paddle.trunc_

### CODE REVIEW

**Review by chenwhql — APPROVED** [review]

File: python/paddle/nn/functional/__init__.py
Comment (sunzhongkai588) [inline python/paddle/nn/functional/__init__.py:1]: functional目录下的inpalce版api文档，都没有引用成功。后续请补一下api_label

**Review by sunzhongkai588 — APPROVED** [review]
LGTM，这个先合，后续还需要做：

- api_label 完善，是paddle.nn.functional下的inpalce文档能正确引用
- 提供inplace api对应的中文文档

**Comment by jeff41404** [comment]
we need to describe in PR title which inplace APIs will be add, it is better to use `[add inplace API] paddle.log_/log2_, ...`

**Review by jeff41404 — APPROVED** [review]
LGTM

**Pull request merged**